From: Sandro Tosi <morph@debian.org>
Date: Sun, 6 May 2018 15:35:38 -0400
Subject: fix segfaults due to wrong systemd service file

The service file has the following line:
User=transmission

It should be replaced with:
User=debian-transmission

Bug-Debian: https://bugs.debian.org/718624
Origin: other, https://bugs.debian.org/718624
Forwarded: no
Last-Update: 2014-10-16
---
 daemon/transmission-daemon.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/transmission-daemon.service b/daemon/transmission-daemon.service
index eaf3d85..e0d1224 100644
--- a/daemon/transmission-daemon.service
+++ b/daemon/transmission-daemon.service
@@ -4,7 +4,7 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-User=transmission
+User=debian-transmission
 Type=notify
 ExecStart=/usr/bin/transmission-daemon -f --log-level=error
 ExecReload=/bin/kill -s HUP $MAINPID
